### PR TITLE
[ZH-TW/EN/ZH] Update CMT_4K/2017

### PR DESCRIPTION
**Pull request by SiderealArt**

### CODE REVIEW

**Comment by TicClick** [comment]
is there any extra work left that keeps the PR a draft?

**Comment by SiderealArt** [comment]
yes, actually I was just going to fix some wording and punctuation, but the check failed because of the beatmap URL, should I fix it or it's fine to leave them?

**Comment by TicClick** [comment]
please do fix them! should be straightforward to do (URLs + extra blank lines if I'm sight-reading it correctly)

**Comment by SiderealArt** [comment]
i've fixed most of the links, but some beatmap seems to be deleted. So I didn't change them.
And the OneDrive link are dead too.

**Comment by TicClick** [comment]
well, nothing we can do about them, other than fix the link format (which I did)

**Comment by TicClick** [comment]
if you don't expect visitors, I, or someone else, can merge this right away -- the pull's been sitting here for slightly more than a month already.

**Comment by SiderealArt** [comment]
uhh.. I'm not sure what you mean. can you rephrase that again? sorry.

**Comment by TicClick** [comment]
we usually merge pull requests that are open for longer than two weeks without waiting for anyone to review them. if you don't have a Chinese-speaking person to check your changes, and don't want to wait either, your changes can be added to wiki without the review step

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
actually looking at the changes I don't see any major blocks of text, only beatmap links and headings, so I guess this can go as-is without `no_native_review`
